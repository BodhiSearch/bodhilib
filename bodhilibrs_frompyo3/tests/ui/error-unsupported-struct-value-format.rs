use frompyo3::FromPyO3;

#[derive(FromPyO3)]
#[frompyo3(name = Resource)]
struct TestUnsupportedStructValueLiteral {}

fn main() {}
