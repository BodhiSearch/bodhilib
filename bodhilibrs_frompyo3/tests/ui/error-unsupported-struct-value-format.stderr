error: expected string literal
 --> tests/ui/error-unsupported-struct-value-format.rs:4:19
  |
4 | #[frompyo3(name = Resource)]
  |                   ^^^^^^^^
